ROADMAP.md

```diff
@@ -8,73 +8,118 @@ not listed now does not mean it will not be considered for Antrea. We definitely
-application protocols (HTTP, DNS, etc.), and extending Antrea diagnostics and
-observability features to get into application level visibility.
-
-* **Multi-cluster networking**
-We would extend Antrea from CNI of a single Kubernetes cluster to multi-cluster
-networking, and implement multi-cluster features like multi-cluster Services,
-cross-cluster connectivity, multi-cluster NetworkPolicies. Antrea multi-cluster
-functionalities are under active development. Check the [Antrea Multi-cluster
-user guide](docs/multicluster/user-guide.md) to learn what features are already
-supported.
-
-* **Analytics**
-With the network flows exported by Antrea, we plan to further build an analytics
-solution that consumes the network flows, and provides traffic analysis,
-NetworkPolicy recommendation, security and network performance monitoring. We
-already started a sub-project of Antrea - [Theia](https://github.com/antrea-io/theia) -
-for network flow visibility and analytics. Stay tunned!
-
-* **K8s Node security**
-So far Antrea focuses on K8s Pod networking and security, but we would like to
-extend Antrea-native NetworkPolicies to cover protection of K8s Nodes too.
-
-* **NetworkPolicy scale and performance tests**
-Evaluate and benchmark the NetworkPolicy implementation performance at a large
-scale, including the policy computation performance of Antrea Controller and the
-OVS datapath performance.
-
-* **OVS with DPDK or AF_XDP**
-Leverage OVS with DPDK or AF_XDP for high performance.
+## Roadmap Items
+
+### Antrea v2
+
+Antrea [version 2](https://github.com/antrea-io/antrea/issues/4832) is coming in
+2024. We are graduating some popular features to Beta or GA, deprecating some
+legacy APIs, dropping support for old K8s versions (< 1.19) to improve support
+for newer ones, and more! This is a big milestone for the project, stay tuned!
+
+### K8s Node security
+
+So far Antrea has focused on K8s Pod networking and security, but we would like
+to extend Antrea-native NetworkPolicies to cover protection of K8s Nodes
+too. There is ongoing work for this, so expect this feature very soon!
+
+### Quality of life improvements for installation and upgrade
+
+We have a few things planned to improve basic usability:
+
+* provide separate container images for the Agent and Controller: this will
+  reduce image size and speed up deployment of new Antrea versions.
+* support for installation and upgrade using the antctl CLI: this will provide
+  an alternative installation method and antctl will ensure that Antrea
+  components are upgraded in the right order to minimize workload disruption.
+* CLI tools to facilitate migration from another CNI: we will take care of
+  provisioning the correct network resources for your existing workloads.
+
+### Core networking features
+
+We are currently working on supporting VLAN tagging for Egress traffic. In the
+long term, we plan to add BGP support to the Antrea Agent, as it is a much
+requested feature.
+
```

Comment:
@antoninbas , Do we support pods in different subnets and vlan tagged(different vlans) traffic in IPAM to communicate to each other

Reply:
We do, but [flexible IPAM](https://github.com/antrea-io/antrea/blob/main/docs/antrea-ipam.md) is orthogonal to this (VLAN tagging for Egress)